src/ec/packages/ec-component-table/table.test.js

```diff
@@ -70,6 +70,15 @@ describe('EC - Table', () => {
 
       return expect(render(withRowExtraClasses)).resolves.toMatchSnapshot();
     });
+
+    test('with missing input data and debug enabled shows the right warning message.', () => {
```

Comment:
You have written logic for 2 main cases: 1) wrong headers 2) wrong rows
Only the first case is covered.

Reply:
yep, it's actually three cases, i can only test two of them in one shot, i added the second :)